tests/core/providers/test_websocket_provider.py

```diff
@@ -18,15 +23,6 @@
     WebsocketProvider,
 )
 
-if sys.version_info >= (3, 8):
```

Comment:
Hmm. strange. The tests were failing without this change. I had even checked the documentation and I couldn't find `concurrent.futures.TimeoutError`. But I just checked and it seems like it magically appears again.

Reply:
Yeah, not sure what happened here. I just noticed that 3.8 tests were breaking so I looked at it and saw this. Tests are green now though 👍